src/cminx/documentation_types.py

```diff
@@ -62,12 +62,25 @@ class DocumentationType(ABC):
     doc: str
     """The full doc comment, cleaned of # characters."""
 
-    @abstractmethod
     def process(self, writer: RSTWriter) -> None:
         """
         Processes the data stored in this documentation type,
         converting it to RST form by using the given RST writer.
 
+        :param writer: RSTWriter instance that the documentation
+                       will be added to.
+        """
+        if ':cminx: no-doc' in self.doc:
+            # self.logger.info('Skipping documentation for function {function_name} because :cminx: no-doc was specified')
```

Comment:
Needs to be removed before merging. No way to access the logger instance without changing the signature but IMO this would be helpful debug-level information.

Reply:
If checking for `:cminx: no-doc` is the way you decide to go forward, you could change this method to a `preprocess()` method that can be overridden by subclasses. This way, you could override it in the function/method processor to provide more detailed information, such as the name of the function that was skipped.